int64 center_x
int64 center_y
int64 height
int64 width
float64 score
string id
int64 distance
